thirdparty/patchmatchnet/README.md

```diff
@@ -0,0 +1,110 @@
+# PatchmatchNet (CVPR2021 Oral)
+official source code of paper 'PatchmatchNet: Learned Multi-View Patchmatch Stereo'
+![](imgs/structure_teaser.jpg)
+
+## Introduction
+PatchmatchNet is a novel cascade formulation of learning-based Patchmatch which aims at decreasing memory consumption and computation time for high-resolution multi-view stereo. If you find this project useful for your research, please cite: 
+
+```
+@misc{wang2020patchmatchnet,
+      title={PatchmatchNet: Learned Multi-View Patchmatch Stereo}, 
+      author={Fangjinhua Wang and Silvano Galliani and Christoph Vogel and Pablo Speciale and Marc Pollefeys},
+      journal={CVPR},
+      year={2021}
+}
+```
+
+## Installation
+### Requirements
+* python 3.7
+* CUDA >= 10.1
+
+```
+pip install -r requirements.txt
+```
+
+## Reproducing Results
+* Download our pre-processed dataset: [DTU's evaluation set](https://drive.google.com/file/d/1jN8yEQX0a-S22XwUjISM8xSJD39pFLL_/view?usp=sharing), [Tanks & Temples](https://drive.google.com/file/d/1gAfmeoGNEFl9dL4QcAU4kF0BAyTd-r8Z/view?usp=sharing) and [ETH3D benchmark](https://polybox.ethz.ch/index.php/s/pmTGWobErOnhEg0). Each dataset is already organized as follows:
+```
+root_directory
+├──scan1 (scene_name1)
+├──scan2 (scene_name2) 
+      ├── images                 
+      │   ├── 00000000.jpg       
+      │   ├── 00000001.jpg       
+      │   └── ...                
+      ├── cams_1                   
+      │   ├── 00000000_cam.txt   
+      │   ├── 00000001_cam.txt   
+      │   └── ...                
+      └── pair.txt  
+```
+Camera file ``cam.txt`` stores the camera parameters, which includes extrinsic, intrinsic, minimum depth and maximum depth:
```

Comment:
do we need to include these details about `cam.txt`? i doubt we will be using it?

Reply:
Yeah, we don't. Actually, the training and evaluation codes are also not included. Shall we mention these in README?

Reply:
I dont think we need to